doc/user-manual/language/instance-arguments.lagda.rst

```diff
@@ -439,6 +439,36 @@ must be implicit, indicating that it needs to be inferred by
 unification whenever the ``B`` instance is used. See
 :ref:`instance-resolution` below for more details.
 
+.. _overlapping-instances:
+
+Overlapping instances
++++++++++++++++++++++
+
+By default, Agda does not allow overlapping instances. Two instances
+are defined to overlap if they could both solve the instance goal
+when given appropriate solutions for their recursive (instance)
+arguments.
+
```

Comment:
It might be worth pointing out that, even if overlapping instances are allowed, Agda will not make unforced choices.